CHANGELOG.md

```
@@ -1,6 +1,8 @@
 # 7.6.2 (in progress)
 
-* Your contribution here!
+##### Bug Fixes and Improvements
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h5

(MD001, heading-increment)